VTEX - Orders API (PII compliant).json

```diff
@@ -1443,8 +1443,7 @@
r\nPCI Connector's Transaction ID (TID):\r\n\r\n    `/api/oms/pvt/orders?searchField=pci_tid&pci_tid=7032909234899834298423209`\r\n\r\n\r\n    \r\nPayment ID (PID):\r\n    \r\n    `/api/oms/pvt/orders?searchField=paymentId&paymentId=2`\r\n\r\n\r\n    \r\nConnector's NSU:\r\n    \r\n    `/api/oms/pvt/orders?searchField=nsu&nsu=2437281`\r\n\\n\\r\\n\\r## Response objects\\n\\r\\n\\r,\r\n**Response object has the following properties:**\r\n\r\n\r\n| Attribute    | Type        | Description |\r\n| --------------- |:---------:| --------------------------------------:|\r\n| `list` | object | Order List Object |\r\n| `shippingEstimateDate` | string | Estimate Shipping Date |\r\n| `affiliateId` | string | Seller Name who was responsible for the order |\r\n| `authorizedDate` | string | Authorized Order Date |\r\n| `callCenterOperatorData` | string | Call Center Operator responsible for the order |\r\n| `name` | string | Client Name|\r\n| `creationDate` | string | Order Creation Date |\r\n| `currencyCode` | string | Currency Code in ISO 4217  |\r\n| `items` | object | Obsolete Field  |\r\n| `lastMessageUnread` | string | Last sent transactional message  |\r\n| `listId` | string | Releated Gift List Id |\r\n| `listType` | string | Releated Gift List Type |\r\n| `marketplaceOrderId` | string | Marketplace Order Id |\r\n| `orderId` | string | Change receipt order Id |\r\n| `orderIsComplete` | boolean | If is a Order Completed |\r\n| `origin` | string | Order Origin: \"Marketplace\" or \"Fulfillment\" |\r\n| `paymentNames` | string | Payment System Name |\r\n| `salesChannel` | string | Order Sales Channel Id |\r\n| `sequence` | string | Order Sequence ID |\r\n| `status` | string | Order Status |\r\n| `statusDescription` | string | Status Description |\r\n| `totalItems` | integer | Total Order Items |\r\n| `totalValue` | integer | Total Value Amount |\r\n| `workflowInErrorState` | boolean | If is a Work Flow Error |\r\n| `workflowInRetry` | boolean | If is in a Work Flow Retry |\r\n| `paging` | object | Paging Details Object |\r\n| `pages` | integer | Paging Total Pages |\r\n| `perPage` | integer | Paging total per Page |\r\n| `total` | integer | Total Result |\r\n| `stats` | object | Results segmented by `totalItems` and `totalValue` |\r\n| `totalItems` | integer | Total Order Items segmeted by `currencyCode` and `origin`|\r\n| `totalValue` | integer | Total Value Amount segmeted by `currencyCode` and `origin`|\r\n| `currencyCode` | string | Currency Code in ISO 4217  |\r\n| `origin` | string | Order Origin: \"Marketplace\" or \"Fulfillment\" |\r\n| `Count` | integer | Orders Count |\r\n| `Facets` | object | Grouping Object |\r\n| `Max` | object | Paging Details Object |\r\n| `Mean` | object | Paging Details Object |\r\n| `Min` | object | Paging Details Object |\r\n| `Missing` | object | Paging Details Object |\r\n| `StdDev` | object | Paging Details Object |\r\n| `Sum` | object | Paging Details Object |\r\n| `SumOfSquares` | object | Paging Details Object |",
```

Comment:
```suggestion
                "description": "Retrieve a list of orders according to the filters described below.\n\r\n\r> This endpoint returns masked order data.\n\r\n\r> This should **not** be used for integrations. Use the [orders Feed or hook](https://developers.vtex.com/vtex-rest-api/docs/feed-v3-1) for this purpose.\n\r\n\rThis endpoint returns only orders that already have been indexed, which takes aproximately four minutes. Because of this, the data retrieved may present inconsistencies. To get live up to date information and [build order integrations](https://developers.vtex.com/vtex-rest-api/docs/erp-integration-set-up-order-integration) use the [orders Feed or hook](https://developers.vtex.com/vtex-rest-api/docs/feed-v3-1).\n\r\n\r> Check the new [Orders onboarding guide](https://developers.vtex.com/vtex-rest-api/docs/orders-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Orders and is organized by focusing on the developer's journey.\n\r\n\r## Do not use request for integrations\n\r> This endpoint returns only orders that already have been indexed, which takes aproximately four minutes. Because of this, the data retrieved may present inconsistencies or become unavailable in periods of peak order volume. In other words, **order integrations built with the List orders endpoint are not reliable** and **will not be supported**. To get live up to date information and [build order integrations](https://developers.vtex.com/vtex-rest-api/docs/erp-integration-set-up-order-integration) use the [orders Feed or hook](https://developers.vtex.com/vtex-rest-api/docs/feed-v3-1).\n\r\n\r## Throtling\n\rEach account can make up to 5000 requests per minute.\n\r\n\r## Request parameters allowed\n\r\",\r\n\"operationId\": \"ListOrdersPiiCompliant| Attribute    | Type        | Description |\r\n| --------------- |:---------:| -------------------------------------------------------------------------------------------:|\r\n| `orderBy` | string | Order Field and Order Type concatenated `orderBy={{OrderField}},{{OrderType}}`  |\r\n| `OrderField` | string | Order Field expected values: `creationDate`,`orderId`,`items`,`totalValue` and `origin` |\r\n| `OrderType` | string | Order Type expected values: `asc` and `desc`  |\r\n\r\n\r\n## Sorting Examples\r\n\r\n\r\nCreationDate:\r\n\r\n    `/api/oms/pvt/orders?orderBy=creationDate,desc`\r\n\r\n\r\nOrderID:\r\n\r\n    `/api/oms/pvt/orders?orderBy=orderId,desc`\r\n\r\n\r\nItems:\r\n\r\n    `/api/oms/pvt/orders?orderBy=items,asc`\r\n\r\n\r\nTotalValue:\r\n\r\n    `/api/oms/pvt/orders?orderBy=totalValue,desc`\r\n\r\n\r\nOrigin: \r\n\r\n    `/api/oms/pvt/orders?orderBy=origin,asc`\r\n\r\n\r\n\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `page` | integer | Page Number  |\r\n\r\n## Pagination Examples:\r\n\r\n    `/api/oms/pvt/orders?page=3`\r\n\r\n\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `per_page` | integer | Quantity orders per page  |\r\n\r\nQuantity per Page Examples:\r\n\r\n    `/api/oms/pvt/orders?per_page=15`\r\n\r\n\r\n\r\n\r\n\r\n<div class=\"alert alert-info\">Pagination Limit: The limit of pages that can be requested is 30.\r\n</div>\r\n\r\n\r\n\r\n## Time zone query \r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `utc` | integer | Time Zone. By including this parameter alone in your query, the call will not convert the UTC nor show the number of orders set for that UTC. To filter orders with the desired UTC, you must also modify date and time in the `f_creationDate=creationDate:` parameter.  |\r\n\r\nTime Zone Example:\r\n\r\n    `/api/oms/pvt/orders?utc=-0200`\r\n\r\n\r\n\r\n\r\n## Request filters \r\n\r\n\r\n## Fulltext\r\n\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `q` | string | Fulltext accepts Order Ids, Client E-mail, Client Document and Client Name  |\r\n\r\n> The `+` caracter isn't allowed in Fulltext Search\r\n\r\n\r\nFulltext filter Examples:\r\n\r\nOrderID: \r\n    \r\n    `/api/oms/pvt/orders?q=v212333lux-02`\r\n\r\n\r\nEmail:\r\n    \r\n    `/api/oms/pvt/orders?q=rodrigo.cunha@vtex.com`\r\n\r\n\r\nDocument: \r\n    \r\n    `/api/oms/pvt/orders?q=21133355524`\r\n\r\n\r\nClientName:\r\n\r\n    `/api/oms/pvt/orders?q=Cunha`\r\n\r\n\r\n\r\n\r\n## Shipping Estimate\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_shippingEstimate` | string | Concatened value of quantity days and sufix `.days`  |\r\n\r\n\r\n\r\n\r\nShipping Estimate filter Examples:\r\n\r\n\r\nNext 7 days:\r\n\r\n    `/api/oms/pvt/orders?f_shippingEstimate=7.days`\r\n\r\n\r\nTomorrow: \r\n\r\n    `/api/oms/pvt/orders?f_shippingEstimate=1.days`\r\n\r\n\r\nToday:\r\n    \r\n    `/api/oms/pvt/orders?f_shippingEstimate=0.days`\r\n\r\n\r\nLate:\r\n    \r\n    `/api/oms/pvt/orders?f_shippingEstimate=-1.days`\r\n\r\n\r\n\r\n\r\n## Invoiced Date\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_invoicedDate` | string | Concatened value sufix `invoicedDate` and range date in Timestamp format  |\r\n\r\n\r\n\r\nInvoiced Date filter Examples:\r\n\r\n1 Day: \r\n\r\n    `/api/oms/pvt/orders?f_invoicedDate=invoicedDate:[2017-01-01T02:00:00.000Z TO 2017-01-02T01:59:59.999Z]`\r\n\r\n\r\n1 Month: \r\n    \r\n    `/api/oms/pvt/orders?f_invoicedDate=invoicedDate:[2017-01-01T02:00:00.000Z TO 2017-02-01T01:59:59.999Z]`\r\n\r\n\r\n1 Year:\r\n\r\n    `/api/oms/pvt/orders?f_invoicedDate=invoicedDate:[2016-01-01T02:00:00.000Z TO 2017-01-01T01:59:59.999Z]`\r\n\r\n\r\n\r\n## Order Date\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_creationDate` | string | Concatened value suffix `creationDate` and range date in Timestamp format  |\r\n\r\n\r\n\r\nOrder Date filter Examples:\r\n\r\n1 Day:\r\n\r\n    `/api/oms/pvt/orders?f_creationDate=creationDate:[2017-01-01T02:00:00.000Z TO 2017-01-02T01:59:59.999Z]`\r\n\r\n\r\n1 Month:\r\n\r\n    `/api/oms/pvt/orders?f_creationDate=creationDate:[2017-01-01T02:00:00.000Z TO 2017-02-01T01:59:59.999Z]`\r\n\r\n\r\n1 Year:\r\n\r\n    `/api/oms/pvt/orders?f_creationDate=creationDate:[2016-01-01T02:00:00.000Z TO 2017-01-01T01:59:59.999Z]`\r\n\r\n\r\n\r\n\r\n## Authorized Date\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_authorizedDate` | string | Concatened value suffix `authorizedDate` and range date in Timestamp format  |\r\n\r\n\r\n\r\nAuthorized Date filter Examples:\r\n\r\n1 Day:\r\n\r\n    `/api/oms/pvt/orders?f_authorizedDate=authorizedDate:[2017-01-01T02:00:00.000Z TO 2017-01-02T01:59:59.999Z]`\r\n\r\n\r\n1 Month:\r\n\r\n    `/api/oms/pvt/orders?f_authorizedDate=authorizedDate:[2017-01-01T02:00:00.000Z TO 2017-02-01T01:59:59.999Z]`\r\n\r\n\r\n1 Year:\r\n\r\n    `/api/oms/pvt/orders?f_authorizedDate=authorizedDate:[2016-01-01T02:00:00.000Z TO 2017-01-01T01:59:59.999Z]`\r\n\r\n\r\n\r\n\r\n## UTMs\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_UtmSource` | string | UTM Source value  |\r\n\r\n\r\n\r\nChannels filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_UtmSource=buscape_campaign`\r\n\r\n\r\n\r\n\r\n## Seller\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_sellerNames` | string | Seller Name value  |\r\n\r\n\r\n\r\nSeller filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_sellerNames=Fast+Shop`\r\n\r\n\r\n\r\n\r\n## Call Center Operator\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_callCenterOperatorName` | string | Call Center Operator Value  |\r\n\r\n\r\n\r\nCall Center Operator filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_callCenterOperatorName=Operator%20Name`\r\n\r\n\r\n\r\n\r\n## Sales Channel Name\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_salesChannel` | string | Sales Channel Name Value  |\r\n\r\n\r\n\r\nSales Channel Name filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_salesChannel=Main`\r\n\r\n\r\n\r\n\r\n\r\n## Sales Channel ID\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `salesChannelId` | string | Sales Channel ID Value  |\r\n\r\n\r\n\r\nSales Channel ID filter Examples:\r\n\r\n    `/api/oms/pvt/orders?salesChannelId=1`\r\n\r\n\r\n\r\n\r\n\r\n## Affiliate ID\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_affiliateId` | string | Affiliate ID Value  |\r\n\r\n\r\n\r\nAffiliate ID filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_affiliateId=WLM`\r\n\r\n\r\n\r\n\r\n## Order Status \r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_status` | string | Order Status Value  |\r\n\r\n\r\n\r\n| Order Status avaible to filter    |\r\n| --------------------- |\r\n| `waiting-for-sellers-confirmation` | \r\n| `payment-pending` | \r\n| `payment-approved` | \r\n| `ready-for-handling` | \r\n| `handling` | \r\n| `invoiced` | \r\n| `canceled` | \r\n\r\n\r\nOrder Status filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_status=ready-for-handling`\r\n\r\n\r\n\r\n\r\n## Order Status Description\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_statusDescription` | string | Order Status Description Value  |\r\n\r\n\r\n\r\n\r\n\r\n| Order Status Description avaible to filter |\r\n| --------------------- |\r\n| `Aguardando+autorização+para+despachar` | \r\n| `Pagamento+Pendente` | \r\n| `Pagamento+Aprovado` | \r\n| `Pronto+para+o+manuseio` | \r\n| `Preparando+Entrega` | \r\n| `Faturado` | \r\n| `Cancelado` | \r\n\r\nOrder Status Description filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_statusDescription=Pronto+para+o+manuseio`\r\n\r\n\r\n\r\n\r\n\r\n## Order Situation\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `incompleteOrders` | boolean | If is a Incomplete Order  |\r\n\r\n\r\n> know more about [Incomplete Orders in VTEX Help](https://help.vtex.com/en/tutorial/understanding-incomplete-orders)\r\n\r\nOrder Situation filter Examples:\r\n\r\n    `/api/oms/pvt/orders?incompleteOrders=true`\r\n\r\n\r\n\r\n\r\n\r\n## Error Situation\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `incompleteOrders` | boolean | Error Situation Type  |\r\n\r\n\r\n\r\n| Error Situation Values avaible to filter |\r\n| ------- |\r\n| `all` | \r\n| `perm` | \r\n| `temp` |  \r\n\r\n\r\nError Situation filter Examples:\r\n\r\n    `/api/oms/pvt/orders?filterError=all`\r\n\r\n\r\n\r\n\r\n\r\n\r\n## Payment Type Name\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_paymentNames` | string | Payment Type Value  |\r\n\r\n\r\n\r\nPayment Type Name filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_paymentNames=Visa`\r\n\r\n\r\n\r\n\r\n\r\n## Rates and Benefits Name\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `f_RnB` | string | Rates and Benefits Name  |\r\n\r\n\r\n\r\nRates and Benefits filter Examples:\r\n\r\n    `/api/oms/pvt/orders?f_RnB=\"Free+Shipping\"`\r\n\r\n\r\n\r\n\r\n## Search Field\r\n\r\n| Attribute    | Type      | Description |\r\n| ------------ |:---------:| -----------:|\r\n| `searchField` | string | Affiliate ID Value  |\r\n| `sku_Ids&sku_Ids` | integer | SKU ID  |\r\n| `listId&listId` | integer | Gift List ID  |\r\n| `tid&tid` | integer | Transaction ID (TID)  |\r\n| `pci_tid&pci_tid` | integer | PCI Connector's Transaction ID (TID)  |\r\n| `paymentId&paymentId` | integer | Payment ID (PID)  |\r\n| `nsu&nsu` | integer | Connector's NSU  |\r\n\r\n\r\n\r\nSearch Field filter Examples:\r\n\r\nSKU ID:\r\n    \r\n    `/api/oms/pvt/orders?searchField=sku_Ids&sku_Ids=11223`\r\n\r\n\r\n\r\nGift List ID:\r\n    \r\n    `/api/oms/pvt/orders?searchField=listId&listId=11223`\r\n\r\n\r\n    \r\nTransaction ID (TID):\r\n    \r\n    `/api/oms/pvt/orders?searchField=tid&tid=54546300238810034995829230012`\r\n\r\n\r\n    \r\nPCI Connector's Transaction ID (TID):\r\n\r\n    `/api/oms/pvt/orders?searchField=pci_tid&pci_tid=7032909234899834298423209`\r\n\r\n\r\n    \r\nPayment ID (PID):\r\n    \r\n    `/api/oms/pvt/orders?searchField=paymentId&paymentId=2`\r\n\r\n\r\n    \r\nConnector's NSU:\r\n    \r\n    `/api/oms/pvt/orders?searchField=nsu&nsu=2437281`\r\n\\n\\r\\n\\r## Response objects\\n\\r\\n\\r,\r\n**Response object has the following properties:**\r\n\r\n\r\n| Attribute    | Type        | Description |\r\n| --------------- |:---------:| --------------------------------------:|\r\n| `list` | object | Order List Object |\r\n| `shippingEstimateDate` | string | Estimate Shipping Date |\r\n| `affiliateId` | string | Seller Name who was responsible for the order |\r\n| `authorizedDate` | string | Authorized Order Date |\r\n| `callCenterOperatorData` | string | Call Center Operator responsible for the order |\r\n| `name` | string | Client Name|\r\n| `creationDate` | string | Order Creation Date |\r\n| `currencyCode` | string | Currency Code in ISO 4217  |\r\n| `items` | object | Obsolete Field  |\r\n| `lastMessageUnread` | string | Last sent transactional message  |\r\n| `listId` | string | Releated Gift List Id |\r\n| `listType` | string | Releated Gift List Type |\r\n| `marketplaceOrderId` | string | Marketplace Order Id |\r\n| `orderId` | string | Change receipt order Id |\r\n| `orderIsComplete` | boolean | If is a Order Completed |\r\n| `origin` | string | Order Origin: \"Marketplace\" or \"Fulfillment\" |\r\n| `paymentNames` | string | Payment System Name |\r\n| `salesChannel` | string | Order Sales Channel Id |\r\n| `sequence` | string | Order Sequence ID |\r\n| `status` | string | Order Status |\r\n| `statusDescription` | string | Status Description |\r\n| `totalItems` | integer | Total Order Items |\r\n| `totalValue` | integer | Total Value Amount |\r\n| `workflowInErrorState` | boolean | If is a Work Flow Error |\r\n| `workflowInRetry` | boolean | If is in a Work Flow Retry |\r\n| `paging` | object | Paging Details Object |\r\n| `pages` | integer | Paging Total Pages |\r\n| `perPage` | integer | Paging total per Page |\r\n| `total` | integer | Total Result |\r\n| `stats` | object | Results segmented by `totalItems` and `totalValue` |\r\n| `totalItems` | integer | Total Order Items segmeted by `currencyCode` and `origin`|\r\n| `totalValue` | integer | Total Value Amount segmeted by `currencyCode` and `origin`|\r\n| `currencyCode` | string | Currency Code in ISO 4217  |\r\n| `origin` | string | Order Origin: \"Marketplace\" or \"Fulfillment\" |\r\n| `Count` | integer | Orders Count |\r\n| `Facets` | object | Grouping Object |\r\n| `Max` | object | Paging Details Object |\r\n| `Mean` | object | Paging Details Object |\r\n| `Min` | object | Paging Details Object |\r\n| `Missing` | object | Paging Details Object |\r\n| `StdDev` | object | Paging Details Object |\r\n| `Sum` | object | Paging Details Object |\r\n| `SumOfSquares` | object | Paging Details Object |",
```